### PR TITLE
added gitlab-ci

### PR DESCRIPTION
**Pull request by JensWehner**
I translated the github-actions into a gitlab-ci as best as I could

### CODE REVIEW

**Comment by JensWehner** [comment]
@bast can you have a look?

File: content/gitlab-ci.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
+# Automatic testing with GitHub Actions
```
Comment (wikfeldt) [inline content/gitlab-ci.md:1]:
```suggestion
# Automatic testing with GitLab CI
```

File: content/gitlab-ci.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+```{figure} img/testing_group_work.png
+:alt: Graph with exercise steps
+:width: 300px
+
+Overview of this exercise. Below we detail the steps.
+```
+
+
+### Step 1: Create a new repository on GitHub
```
Comment (wikfeldt) [inline content/gitlab-ci.md:33]:
```suggestion
### Step 1: Create a new repository on GitLab
```

File: content/gitlab-ci.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+- Then add a LICENSE via a pull/merge request, referencing the issue number.
+```
+
+---
+
+## Where to go from here
+
+- This example was using Python but you can achieve the same automation for R or Fortran or C/C++ or other languages
+- GitHub Actions has a [Marketpace](https://github.com/marketplace?type=actions) which offer wide range of automatic workflows
+- Check out the [tutorial](https://coderefinery.github.io/testing/gh-actions/) about GitHub Actions
```
Comment (wikfeldt) [inline content/gitlab-ci.md:210]: is there a corresponding page for GitLab CI? Maybe just link to the docs: https://docs.gitlab.com/runner/

**Review by wikfeldt — CHANGES_REQUESTED** [review]
fantastic, thanks! We needed this. Found a couple of mentions of GitHub that should be replaced by GitLab, and have a question on a list of CI runner workflows similar to the GH marketplace

File: content/gitlab-ci.md (hunk 3 above)
Comment (JensWehner) [reply content/gitlab-ci.md:210]: Yeah not really it has some default template workflows, but they are a lot more complex, using caching etc.. So I am a bit hesitant to link them here.

**Review by wikfeldt — APPROVED** [review]
looks very good, thanks

**Comment by bast** [comment]
Thanks! What we can try later is to combine the pages into one and let participants switch between "tabs" but this can happen later. Thanks a lot!

**Pull request merged**